repository-data/application/src/main/resources/hcm-content/content/documents/administration/configuration/local-feature---visitscotland.com.yaml

```
@@ -23,52 +23,51 @@
-      events-listings.page-size, form.breg.legal-basis.enable, site.path.banner, cludo.customer-id,
-      cludo.engine-id, cludo.experience-id, feature.products-search.enable, feature.table-of-contents.enable,
-      feature.global-search.enable, form.breg.legal-basis.text, site-id]
+      global-search.path, ski.timeout, site.path.ski-landing, site.path.icentre-landing,
+      site.path.newsletter, site.path.campaigns, form.json.countries, form.json.messages,
+      form.marketo.munchkin, form.maketo.instance-url, form.recaptcha-key, form.marketo.scripts,
+      form.marketo.script, links.download.extensions, events-listings.page-size, form.breg.legal-basis.enable,
+      site.path.banner, cludo.customer-id, cludo.engine-id, cludo.experience-id, feature.products-search.enable,
+      feature.table-of-contents.enable, feature.global-search.enable, form.breg.legal-basis.text,
+      site-id]
     resourcebundle:messages: ['https://community.visitscotland.com/', discussions/tagged/,
       'localhost,www.visitscotland.com,beta.visitscotland.com,blog.visitscotland.com',
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 30 but found 6

(indentation)